lib/orbit-common/cache.js

```diff
@@ -323,7 +321,8 @@ var Cache = Class.extend({
         id = operation.path[1],
         links = _this.schema.models[type].links;
 
-      Object.keys(links).forEach(function(link) {
+      for (var i = 0, linkKeys = Object.keys(links), len = linkKeys.length, link; i < len; ++i) {
```

Comment:
nit: can you unroll the variable initialization steps into the var statement right before this?  having three of them in a single for loop is a readability problem